sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -183,14 +183,19 @@ public void close() {
         final OffsetDateTime now = OffsetDateTime.now();
         Duration initialInterval = Duration.between(now, initialLockedUntil);
 
-        if (initialInterval.isNegative()) {
-            logger.info("Duration was negative. now[{}] lockedUntil[{}]", now, initialLockedUntil);
+        // If remaining time to renew is less than 400ms, renew immediately
+        // Lock renewal to follow the same behavior as C# SDK
+        if (initialInterval.isNegative() || initialInterval.toMillis() < 400) {
+            logger.info("Duration was negative or less than 400ms. now[{}] lockedUntil[{}]", now, initialLockedUntil);
             initialInterval = Duration.ZERO;
         } else {
             // Adjust the interval, so we can buffer time for the time it'll take to refresh.
-            final Duration adjusted = MessageUtils.adjustServerTimeout(initialInterval);
+            final long bufferInMilliSec = Math.min(initialInterval.toMillis() / 2, maxLockRenewalDuration.toMillis());
```

Comment:
same comment - To align with .net, we should use 10 sec as the max cap for buffer time.

Reply:
done! Thanks Anu!